doc/nrf/app_dev/device_guides/nrf70/index.rst

```diff
@@ -110,11 +109,15 @@ The following nRF70 Series shields are available and defined in the :file:`nrf/b
      - ``nrf7002ek/nrf7000``
      - | :ref:`Development guide <ug_nrf7002ek_gs>`
        | `User Guide <nRF7002 EK User Guide_>`_
-   * - nRF7002 :term:`Expansion Board (EB)`
+   * - nRF7002 :term:`Expansion Board (EB)` (Deprecated)
      - PCA63561
      - ``nrf7002eb``, ``nrf7002eb_interposer_p1`` (nRF54 Series)
      - | :ref:`Development guide <ug_nrf7002eb_gs>`
        | `User Guide <nRF7002 EB User Guide_>`_
+   * - nRF7002 :term:`Expansion Board (EB)` II
```

Comment:
I think this linking to EB in glossary is already mentioned in the line 112, so we can remove it from here.
```suggestion
   * - nRF7002-EB II
```